### PR TITLE
pagination

### PR DESCRIPTION
**Pull request by jota12x**
#9

### CODE REVIEW

File: src/containers/SearchResultsPage.tsx [hunk 1]
```diff
@@ -15,39 +17,73 @@ import UrlParams from 'interfaces/UrlParams';
...
+            ? 'paperwithcode'
+            : 'github';
+        setActiveIndex(searchParams.type !== undefined ? searchParams.type : 0);
+        const requestBody = new SearchQuery(
+          currentResultIndex,
+          searchParams.q,
+          source,
+        );
+        if (currentResultIndex === 0) setShowPlaceholder(true);
+        else setShowLoader(true);
```
Comment (rhezaharliman) [inline src/containers/SearchResultsPage.tsx]: Sorry for becoming to detail, I think it'll be easier to read like (Sorry if it's not JS writing convention 😅 )
```
if (currentResultIndex ===0) {
  setShowPlaceholder(true);
} else {
  setShowLoader(true);
}
```

Comment (jota12x) [reply src/containers/SearchResultsPage.tsx]: LGTM!

Comment (jota12x) [reply src/containers/SearchResultsPage.tsx]: Solved!

Comment (rhezaharliman) [reply src/containers/SearchResultsPage.tsx]: Perfect!

**Pull request merged**